GMG/Scenes/Home/HomeIntent.swift

```diff
@@ -129,4 +134,16 @@ final class HomeIntent: HomeIntentProtocol {
             Logger.error(String(describing: error))
         }
     }
+
+    func selectLastScore(_ scores: [Score]) {
+        guard let model else { return }
+
+        if scores.isEmpty {
+            model.setSelectedScore(nil)
+            return
+        }
+
+        let last = scores.last!
+        model.setSelectedScore(last)
```

Comment:
강제 옵셔널 언래핑 사용은 지양해주세요~

```suggestion
        guard let lastScore: Score = scores.last else {
            model.setSelectedScore(nil)
            return
        }

        model.setSelectedScore(lastScore)
```

Reply:
이렇게도 가능해요.

```suggestion
        let lastScore: Score? = scores.last
        model.setSelectedScore(lastScore)
```